packages/frontend/src/redux/slices/availableAccounts/index.js

```diff
@@ -19,22 +20,13 @@ const availableAccountsSlice = createSlice({
     name: SLICE_NAME,
     initialState,
     extraReducers: ((builder) => {
-        builder.addCase(refreshAccountOwner.pending, (state) => {
-            set(state, ['status', 'loading'], true);
-            set(state, ['status', 'error'], initialErrorState);
-        });
         builder.addCase(refreshAccountOwner.fulfilled, (state, action) => {
-            set(state, ['status', 'loading'], false);
-            set(state, ['status', 'error'], initialErrorState);
-
             set(state, ['items'], Object.keys((action.payload && action.payload.accounts) || {}).sort());
         });
-        builder.addCase(refreshAccountOwner.rejected, (state, { error }) => {
-            set(state, ['status', 'loading'], false);
-            set(state, ['status', 'error'], {
-                message: error?.message || 'An error was encountered.',
-                code: error?.code
-            });
+        handleAsyncThunkStatus({
+            asyncThunk: `refreshAccountOwner`,
+            buildStatusPath: ({ meta: { arg: { accountId }}}) => ['status'],
```

Comment:
Would it make sense to omit the arrow function parameters to make it more clear that the path is static?

Reply:
Yup it needs to stay a function but since it doesn't actually need anything from the action it can take zero parameters. It makes typing a little wonky but it prevents `accountId` from being an unreferenced variable.

Reply:
Ahh I see, I misunderstood. Done.